bpython/repl.py

```diff
@@ -817,11 +820,11 @@ def write2file(self):
                 self.interact.notify(_('Save cancelled.'))
                 return
 
-        s = self.formatforfile(self.getstdout())
+        stdout_text = self.formatforfile(self.getstdout())
```

Comment:
This isn't just stdout text, it's session text; the interleaving of stdin, stdout, and stderr.